tests/helpers/preload-assets.js

```diff
@@ -0,0 +1,26 @@
+import Ember from 'ember';
+import AssetLoader from 'ember-asset-loader/services/asset-loader';
```

Comment:
We should use `addon-test-support` for any publicly exposed test helpers (not sure if this one is meant for just us in the addon or for all users).